### PR TITLE
homework 10

### PR DESCRIPTION
**Pull request by userhost1**
check please.

### CODE REVIEW

File: src/homework10/Part3T30.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+        for (int i = 0; i < numbers1.size() - 1; i++) {
+            for (int j = 1; j < numbers1.size(); j++) {
+                if (numbers1.get(i).equals(numbers1.get(j))) {
+                    numbers1.remove(j);
+                }
+            }
+        }
+        Collections.sort(numbers1);
+        System.out.println("ArrayList [after removing] " + numbers1);
```
Comment (TheFallen058) [inline src/homework10/Part3T30.java:34]: ArrayList [before removing] [6, 6, 4, 4, 7, 4, 6, 1, 4, 3, 5, 4, 2, 6, 3, 5, 1, 9, 6, 3, 3, 3, 3, 3, 4, 1, 5, 4, 10, 3]
TreeSet [before removing] [1, 2, 3, 4, 6, 7, 8, 9, 10]
ArrayList [after removing] [1, 3, 3, 4, 5, 6, 10]
TreeSet [after removing] [1, 2, 3, 4, 6, 7, 8, 9, 10]
I got this result, please check ArrayList [after removing]. It contains duplicates. Logic is incorrect, check solution from previous task as tip

**Review by TheFallen058 — CHANGES_REQUESTED** [review]
Please fix task 30

File: src/homework10/Part3T30.java (hunk 1 above)
Comment (userhost1) [reply src/homework10/Part3T30.java:34]: yeap. my mistake.
fixed

Comment (userhost1) [reply src/homework10/Part3T30.java:34]: please check library

**Review by TheFallen058 — APPROVED** [review]
done

**Comment by userhost1** [comment]
thanks

**Pull request merged**